### ISSUE TITLE
Cannot find ctags executable in arch linux

### ISSUE THREAD

**Issue by opensourcegeek** [dan-t/rusty-tags#58]
I have got ctags installed, but for some reason it's printing the following error when I try to generate tags by running `rusty-tags vi`. Any idea why this might happen?

```
Couldn't find 'ctags' executable! Searched for executables with names: [""]. Is 'ctags' correctly installed?
```

**Comment by opensourcegeek** [comment]
Found the issue - I used default config file which left ctags_exe empty

**Comment by dan-t** [comment]
On Wed, Apr 03, 2019 at 07:02:45PM +0000, opensourcegeek wrote:
> Found the issue - I used default config file which left ctags_exe empty

I consider this to be a bug which is now fixed in 'rusty-tags 3.5.1'.

Greetings,
Daniel
